.circleci/config.yml

```diff
@@ -130,7 +130,30 @@ jobs:
 
   test:
     description: |
-      Run tests against given version of JDK and Clojure
+      Run quick tests against given version of JDK and Clojure without andersonizing dependencies
```

Comment:
andersonizing -> inlining :-)

Let's not get too specific with the particular tool.